### ISSUE TITLE
Consider using [!NOTE] blockquote for browser/platform compatibility notes

### ISSUE THREAD

**Issue by jonasfj** [google/webcrypto.dart#115]
In: https://pub.dev/documentation/webcrypto/latest/webcrypto/AesCtrSecretKey/encryptBytes.html

And a few other places we write: `**Remark** ...` to indicate that a particular browser or platform doesn't support a particular feature.

Instead of using this form, it'd be cool to use the [`[!NOTE]` format](https://github.com/orgs/community/discussions/16925).

**Example**

> [!NOTE]
> Remark Firefox does not implement counter rollover for AES-CTR correctly. Picking a sufficiently large `length` and using a `counter` that isn't filled with 0xff will likely avoid counter rollovers. See [bug 1803105](https://bugzilla.mozilla.org/show_bug.cgi?id=1803105) for details.

**Markdown**

```md
> [!NOTE]
> Remark Firefox does not implement counter rollover for AES-CTR correctly. Picking a sufficiently large `length` and using a `counter` that isn't filled with 0xff will likely avoid counter rollovers. See [bug 1803105](https://bugzilla.mozilla.org/show_bug.cgi?id=1803105) for details.
```

---------------------

I think this should work with recent versions of `dartdoc` too! :rocket: 

We should test it out locally first, and we should make sure to migrate all such remarks in one PR.

**Comment by TropicolX** [comment]
Hi @jonasfj. Can I be assigned to this?

**Comment by jonasfj** [comment]
Done,

**Comment by jonasfj** [comment]
There is a thread about this package in the #packages channel on dart_community

https://dart.dev/community

Feel free to join and say hi.

**Comment by TropicolX** [comment]
Alright, thanks!
Will do

**Comment by TropicolX** [comment]
Hi @jonasfj,
How do I test out the changes locally? Can I get a preview UI similar to this: https://pub.dev/documentation/webcrypto/latest/webcrypto/AesCtrSecretKey/encryptBytes.html

**Comment by jonasfj** [comment]
Run `dartdoc`

See: https://pub.dev/packages/dartdoc

**Comment by TropicolX** [comment]
@jonasfj Changed all "Remark" notes to use the [!NOTE] format.

1. https://pub.dev/documentation/webcrypto/latest/webcrypto/AesCtrSecretKey/encryptBytes.html, https://pub.dev/documentation/webcrypto/latest/webcrypto/AesCtrSecretKey/decryptBytes.html, https://pub.dev/documentation/webcrypto/latest/webcrypto/AesCtrSecretKey/encryptStream.html,
https://pub.dev/documentation/webcrypto/latest/webcrypto/AesCtrSecretKey/decryptStream.html
![image](https://github.com/google/webcrypto.dart/assets/68024640/fd62a229-9364-4332-822f-59fd8f548fdc)
![image](https://github.com/google/webcrypto.dart/assets/68024640/d2c03672-1f9e-4fe9-b535-e4d2f9b816c8)

2. https://pub.dev/documentation/webcrypto/latest/webcrypto/EllipticCurve.html
![image](https://github.com/google/webcrypto.dart/assets/68024640/2e6d974c-ee9d-4d90-99b6-a7c17718ad45)
![image](https://github.com/google/webcrypto.dart/assets/68024640/2fe6f18f-6fbf-4a19-8032-6a0d32da2efe)

3. https://pub.dev/documentation/webcrypto/latest/webcrypto/AesGcmSecretKey/encryptBytes.html
![image](https://github.com/google/webcrypto.dart/assets/68024640/f1c1c40f-2b08-488b-9194-a47db420aa25)
![image](https://github.com/google/webcrypto.dart/assets/68024640/5a2d9d0a-21ad-403a-8386-50d3157255db)

If this works, I can proceed to create a PR.

**Comment by TropicolX** [comment]
Hi @jonasfj, thanks for the review.
I've created a PR for the issue here #126 

----

Regarding the enum, I have no idea how to add a private member to an enum in dart 😅 but I'd love to look into it. Any pointers on how to do this would also be appreciated.

**Comment by jonasfj** [comment]
Fixed in https://github.com/google/webcrypto.dart/pull/126

--------

@TropicolX pro tip: If you write `Fixed #115` in the description of a PR, then it'll close the associated issue when the PR is landed.
